pkg/controller/utils/utils.go

```diff
@@ -184,17 +184,11 @@ func CheckLicenseKey(ctx context.Context, cli client.Client) error {
 	return cli.Get(ctx, DefaultInstanceKey, instance)
 }
 
-// ValidateCertPair validates the cert pair secret in the tigera-operator namespace.
-func ValidateCertPair(client client.Client, certPairSecretName, keyName, certName string) (*corev1.Secret, error) {
-	return ValidateCertPairInNamespace(client, render.OperatorNamespace(), certPairSecretName, keyName, certName)
-}
-
 // ValidateCertPairInNamespace checks if the given secret exists in the given
 // namespace and if so that it contains key and cert fields. If a secret exists then it is returned.
 // If there is an error accessing the secret (except NotFound) or the cert
 // does not have both a key and cert field then an appropriate error is returned.
 // If no secret exists then nil, nil is returned to represent that no cert is valid.
-// TODO: Replace this with version of ValidateCertPair taking in a ns.
 func ValidateCertPairInNamespace(client client.Client, namespace, certPairSecretName, keyName, certName string) (*corev1.Secret, error) {
```

Comment:
I think you could / maybe should keep the old name, I don't think you need to describe that this is for a namespace when it's the only cert validation function.

Reply:
Good point, fixed!